flixel/addons/editors/pex/FlxPexParser.hx

```diff
@@ -149,7 +150,14 @@ class FlxPexParser
 			maxColor: FlxColor.fromRGBFloat(minR + varR, minG + varG, minB + varB, minA + varA)
 		};
 	}
-	
+
+	private static inline function getNodeName(property:String, config:Fast):String
+	{
+		// for backwards compatibility, check for versions of properties that
```

Comment:
That's one I know of but there were apparently multiple typos over time: https://github.com/Gamua/Starling-Extension-Particle-System/blob/master/src/starling/extensions/PDParticleSystem.as#L260

It seems like it wouldn't hurt to check all nodes. If Fast had a case-insensitive way to resolve nodes that would be even better.